### ISSUE TITLE
Contours Rotating (by the compass) "reset" bug

### ISSUE THREAD

**Issue by RealMook** [CGCookie/script-bakery#106]
Subj.
http://www.pasteall.org/blend/23577 - here is the blend showing what I want to have (I've draw approx. ellipses by GP lines in two projections).
Working with "R" keyboard command seems to be fine. But I use the "compass" now (it's funnier for me) and got this little bug (I hope it's not a feature ;)  )

Try to make one contour then rotate it accordingly to my gr. pencil lines. 1st on the top, 2nd on the front view.
Once 1st is ok, then you rotate (I got that it's capable by arrow without the red line on the compass) 2nd... and it resets the 1st operation.
It looks like it re-projects the contour "killing" the 1st operation. Like it's creating it from scratch.

**Comment by carter2422** [comment]
Do I understand you correctly, rotating one stroke is resetting the orientation of the previous stroke?

**Comment by RealMook** [comment]
No.. it's resetting the previous action for the same stroke. So I want it to be leaned (from one view) and rotated (from another). "R" works as intended here. But (another thing) "R" twists a contour. It removes a twist once I click on it again. Not quick... I suppose it's due to my chaotic manner of contours' creating. I slice the majors 1st and slice minors after them.

**Comment by carter2422** [comment]
Oh got it. This should already be fixed or is on the list to be fixed.

**Comment by patmo141** [comment]
RealMook, can you make a vid or screen shot sequence of this?  I'm having a little trouble visualizing

**Comment by patmo141** [comment]
re-the twisting.  Because it has to derive a contour from the raw underlying mesh, twist is lost because it is re-deriving a totally new contour when you "edge slide."  This is unavoidable without some heavy coding where I would save the last stroke and then align to it but right now it's low priority.  I need to make a post on "best practices" to work around this limitation a bit
